axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -623,63 +623,42 @@
       "networkType": "testnet",
-      "chainType": "stellar",
       "explorer": {
         "name": "Soroban Nownodes",
         "url": "https://soroban.nownodes.io/testnet"
       },
       "tokenSymbol": "XLM",
       "contracts": {
         "axelar_gateway": {
-          "address": "CBGNPCKNCKKKCI5AUOS2K6H4JDLGPFHHXNDNOEZBIA3VO2NBEZQXFAG5",
-          "deployer": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG",
+          "address": "CBECMRORSIPG4XG4CNZILCH233OXYMLCY4GL3GIO4SURSHTKHDAPEOVM",
+          "deployer": "GDUITDF2LI3R5HM4KYRLLNRLEWKYBFVZVOEB6HSL7EOW2KO2LD6V4GPM",
+          "wasmHash": "4143f187c235fe365c547047c06c4fd7fb7b7eff664b9d9b044112d91bfef13c",
           "initializeArgs": {
-            "owner": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG",
-            "operator": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG",
-            "domainSeparator": "e33a8ab3341e36bdcc552924ea42c85ef40dc2cfa41259eb166fb254bc3f036a",
+            "owner": "GDUITDF2LI3R5HM4KYRLLNRLEWKYBFVZVOEB6HSL7EOW2KO2LD6V4GPM",
+            "operator": "GDUITDF2LI3R5HM4KYRLLNRLEWKYBFVZVOEB6HSL7EOW2KO2LD6V4GPM",
+            "domainSeparator": "293407cef6cb084c2c7ca31794d4cc95d8c3ea2aced086aebb518a2c37b28dc6",
             "minimumRotationDelay": "0",
             "previousSignersRetention": "15",
             "initialSigners": [
               {
                 "nonce": "0000000000000000000000000000000000000000000000000000000000000000",
                 "signers": [
                   {
-                    "signer": "168983122cd64add01997a5ad115fbcb52fea5ed659a9702b83efbcb3a5e598e",
+                    "signer": "e8898cba5a371e9d9c5622b5b62b25958096b9ab881f1e4bf91d6d29da58fd5e",
                     "weight": "1"
                   }
                 ],
                 "threshold": "1"
               }
             ]
           }
-        },
-        "axelar_gas_service": {
-          "address": "CDBPOARU5MFSC7ZWXTVPVKDZRHKOPS5RCY2VP2OKOBLCMQM3NKVP6HO7",
-          "deployer": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG",
-          "initializeArgs": {
-            "gasCollector": "CB2LIDEDPDKUXVFHOQ2PHPEJOJBD7YDN43OB3O6D43IYQX4YCPTK4ODI"
-          }
-        },
-        "axelar_operators": {
-          "address": "CB2LIDEDPDKUXVFHOQ2PHPEJOJBD7YDN43OB3O6D43IYQX4YCPTK4ODI",
-          "deployer": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG",
-          "initializeArgs": {
-            "owner": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG"
-          }
-        },
-        "example": {
-          "address": "CDURON7OSAAOWLKIQRNIPFZYLTMEWCRQOZIURAIS5JK3D2GU5IUPNSF6",
-          "deployer": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG",
-          "initializeArgs": {
-            "gatewayAddress": "CBGNPCKNCKKKCI5AUOS2K6H4JDLGPFHHXNDNOEZBIA3VO2NBEZQXFAG5",
-            "gasServiceAddress": "CDBPOARU5MFSC7ZWXTVPVKDZRHKOPS5RCY2VP2OKOBLCMQM3NKVP6HO7"
-          }
```

Comment:
The deployment is incomplete. These other contracts need to be deployed as well, and gas service collector needs to be set to the operators contract

Reply:
Added the missing contracts